zulipterminal/ui_tools/buttons.py

```diff
@@ -268,6 +269,35 @@ def mark_muted(self) -> None:
     # TODO: Handle event-based approach for topic-muting.
 
 
+class EmojiButton(TopButton):
+    def __init__(self, controller: Any, width: int, emoji_name: str,
+                 message: Message) -> None:
+        self.controller = controller
+        self.emoji_name = emoji_name
+        self.message = message
+        super().__init__(
+            controller=controller,
+            caption=emoji_name,
+            prefix_character='',
+            show_function=controller.model.toggle_message_reaction,
+            width=width)
+        if self.user_has_reacted_to_msg():
+            self.update_widget(('bar', ' ✓ '), 'bar')
```

Comment:
I guess the `right` symbol is used somewhere in the stream info as well. It would be nice to refactor it (in a prior commit) in `symols.py` and use the generalized name elsewhere :)